src/core/dbus-execute.c

```diff
@@ -385,7 +385,15 @@ static int property_get_syscall_filter(
                 if (!name)
                         continue;
 
-                if (num >= 0) {
+                if (num == EXEC_SYSCALL_ERROR_NUMBER_KILL) {
+                        s = strjoin(name, ":kill");
+                        if (!s)
+                                return -ENOMEM;
+                } else if (num == EXEC_SYSCALL_ERROR_NUMBER_LOG) {
+                        s = strjoin(name, ":log");
+                        if (!s)
+                                return -ENOMEM;
+                } else if (num >= 0) {
```

Comment:
I'd instead add seccomp_errno_or_action_to_string() or so, that does this conversion, and place it in seccomp-util.h